R/acquaint-package.R

```diff
@@ -5,3 +5,12 @@
 #' @import rlang
 ## usethis namespace: end
 NULL
+
+.onLoad <- function(libname, pkgname) {
+  acquaint_socket <<- switch(
```

Comment:
Should `acquaint_socket` live in `the$`? I don't imagine any specific issues with this double assignment but it's still giving me the heebie jeebies.

Reply:
I've been doing it for yonks in https://github.com/r-lib/mirai/blob/main/R/mirai-package.R#L40-L56 so it doesn't strike me as odd. At this point (pkg onLoad hook), the namespace isn't sealed yet so you can do these things. Note we define `acquaint_socket` elsewhere to ensure it gets assigned to the package namespace.

I can change it to live in `the` if it makes you more comfortable. It's no big deal, except that it'd then be possible to modify... which feels like the less clean option to me.

Reply:
Ok, I've made the switch in [1c4623f](https://github.com/simonpcouch/acquaint/pull/26/commits/1c4623f9767768c57767fd03f26298e5bf57a446), and renamed it `the$socket_url` so it's immediately clear it's a URL rather than an actual socket.

Reply:
I appreciate you making that happen!!